packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -86,7 +86,7 @@ export class Breadcrumb extends BaseComponent<IBreadcrumbProps, IBreadcrumbState
             { renderedItems.map(
               (item, index) => (
                 <li className='ms-Breadcrumb-listItem' key={ item.key || String(index) } ref={ item.key || String(index) }>
-                  { this._renderItem(item) }
+                  { this.props.onRenderItem ? this.props.onRenderItem(item, this._renderItem) : this._renderItem(item) }
```

Comment:
You can simplify this line by doing this:

```tsx
let { onRenderItem = this._onRenderItem } = this.props;
```

This will use the member as the "default" if not provided. Then just use onRenderItem without the ternary.